recipes/hazelcast-cpp-client/all/patches/gcc_11_1_0.patch

```diff
@@ -0,0 +1,40 @@
+diff --git a/hazelcast/include/hazelcast/client/protocol/ClientMessage.h b/hazelcast/include/hazelcast/client/protocol/ClientMessage.h
+index 462ea85b0..f2686506a 100644
+--- a/hazelcast/include/hazelcast/client/protocol/ClientMessage.h
++++ b/hazelcast/include/hazelcast/client/protocol/ClientMessage.h
+@@ -190,6 +190,9 @@ struct HAZELCAST_API is_trivial_entry_vector<
+  */
+ class HAZELCAST_API ClientMessage
+ {
++    template<typename T>
```

Comment:
You should contribute this back upstream to the project

Reply:
Do you mean previous versions at conan or the hazelcast/hazelcast-cpp-client repo ?

Reply:
To hazelcast/hazelcast-cpp-client repo :)